lib/eyaml/railtie.rb

```diff
@@ -30,11 +36,11 @@ def valid?(pathname)
           pathname.exist?
         end
 
-        def secrets_files
+        def secrets_files(secrets_or_credentials)
           EYAML::SUPPORTED_EXTENSIONS.map do |ext|
             [
-              Rails.root.join("config", "secrets.#{ext}"),
-              Rails.root.join("config", "secrets.#{Rails.env}.#{ext}")
+              Rails.root.join("config", "#{secrets_or_credentials}.#{ext}"),
```

Comment:
nit: maybe hint that these files will be read and merged from the least specific to the more specific one? also, should there be, say, `credentials.production.eyaml` and `credentials.eyml` the latter will overwrite everything introduced by `credentials.production.eyaml` even though it is more specific (specific to an environment). So to play this safe it would be better to do the product the other way around - first "envless" variants for all extensions, then "enved" variants for all the extensions. Still a bit magical but will be more logical IMO.